### PR TITLE
new check for correct hostname setting

### PR DESCRIPTION
**Pull request by mpalissa**

### CODE REVIEW

File: scripts/lib/check/0820_hostname.check [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+function check_0820_hostname {
+
+    logTrace "<${BASH_SOURCE[0]}:${FUNCNAME[*]}>"
+
```
Comment (PeterPitterling) [inline scripts/lib/check/0820_hostname.check:6]: missing MODIFICATION SECTION .. e.g.    
    # MODIFICATION SECTION>>
    local -r sapnote_sles='#1944799'
    local -r sapnote_rhel='#2009879'
    # MODIFICATION SECTION<<

File: scripts/lib/check/0820_hostname.check [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    local -i _retval=99
+    local -r sapnote='#611361'
+
+# import short and full hostname
```
Comment (PeterPitterling) [inline scripts/lib/check/0820_hostname.check:10]: indentation !!

File: scripts/lib/check/0820_hostname.check [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+short_hostname=$(hostname)
+full_hostname=$(hostname -f)
+
+# extract short hostname from full hostname
+short_hostname_from_full=$(echo $full_hostname | cut -d'.' -f1)
+
+# Comparison
```
Comment (PeterPitterling) [inline scripts/lib/check/0820_hostname.check:17]: L17-22 should be part of CHECK

File: scripts/lib/check/0820_hostname.check [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+if [ "$short_hostname" = "$short_hostname_from_full" ]; then
+  _retval=0
+  else
+  _retval=66
+fi
+
+if [[ ${_retval} -eq 0 ]]; then
```
Comment (PeterPitterling) [inline scripts/lib/check/0820_hostname.check:24]: # CHECK
    if [[ ${_retval} -eq 99 ]]; then

**Comment by mpalissa** [comment]
added to checksets

**Pull request merged**